code/game/machinery/computer/robot.dm

```diff
@@ -77,24 +77,23 @@
 		return
 
 	switch(action)
-		if("killbot")
-			if(allowed(usr))
-				var/mob/living/silicon/robot/R = locate(params["ref"]) in GLOB.silicon_mobs
-				if(can_control(usr, R) && !..())
-					R.self_destruct(usr)
-			else
-				to_chat(usr, span_danger("Access Denied."))
 		if("stopbot")
 			if(allowed(usr))
 				var/mob/living/silicon/robot/R = locate(params["ref"]) in GLOB.silicon_mobs
 				if(can_control(usr, R) && !..())
-					message_admins(span_notice("[ADMIN_LOOKUPFLW(usr)] [!R.lockcharge ? "locked down" : "released"] [ADMIN_LOOKUPFLW(R)]!"))
-					log_silicon("[key_name(usr)] [!R.lockcharge ? "locked down" : "released"] [key_name(R)]!")
-					log_combat(usr, R, "[!R.lockcharge ? "locked down" : "released"] cyborg")
-					R.SetLockdown(!R.lockcharge)
-					to_chat(R, !R.lockcharge ? span_notice("Your lockdown has been lifted!") : span_alert("You have been locked down!"))
-					if(R.connected_ai)
-						to_chat(R.connected_ai, "[!R.lockcharge ? span_notice("NOTICE - Cyborg lockdown lifted") : span_alert("ALERT - Cyborg lockdown detected")]: <a href='?src=[REF(R.connected_ai)];track=[html_encode(R.name)]'>[R.name]</a><br>")
+					if(isAI(usr)&&R.ailockdown&&R.lockcharge||isAI(usr)&&!R.lockcharge||!isAI(usr))
```

Comment:
space these out also condense the logic a little bit. 
`isAI(usr) && (R.ailockdown && R.lockcharge || !R.lockcharge) || !isAI(usr)`
is better